### ISSUE TITLE
https://github.com/google/spatial-media/releases/tag/v2.0

### ISSUE THREAD

**Issue by swy** [homebysix/recipe-robot#98]
Attempted to use recipe-robot to create autopkg scripts for https://github.com/google/spatial-media/releases/tag/v2.0 .  Blew up the robot with a 
UnicodeEncodeError: 'ascii' codec can't encode character u'\xba' in position 59: ordinal not in range(128)
error.  Looks like it's not handling the unicode degree symbol from the github description(?):

A collection of specifications and tools for 360° video and spatial audio, including:

**Comment by homebysix** [comment]
Interesting — when I ran Recipe Robot on the URL above, I didn't get the unicode error. But I did encounter a different problem: Recipe Robot grabbed the Windows zip instead of the Mac zip.

Here's something you might try:

1. Download the Mac release zip.

2. Run `recipe-robot --verbose /path/to/downloaded.zip`

3. Open the finished recipe and replace the `URLDownloader` processor with a `GitHubReleasesInfoProvider` processor, being sure to specify the `asset_regex` input variable such that the Mac zip is selected. Maybe something like:

    ```
		<dict>
			<key>Arguments</key>
			<dict>
				<key>asset_regex</key>
				<string>.*\.mac\.zip</string>
				<key>github_repo</key>
				<string>google/spatial-media</string>
			</dict>
			<key>Processor</key>
			<string>GitHubReleasesInfoProvider</string>
		</dict>
    ```

4. Verify that the rest of the recipes work, now that the download recipe has been modified.

Let me know how that goes.
